### PR TITLE
feat: add Football World Cup 2022 badge

### LINKED ISSUE #25674: World Cup 2022 Badge
Add a new badge for the `football/world-cup-2022` tag.

World Cup 2022 Badge

Source: https://gist.github.com/HarryFischer/92e792214212555384de36691d25a2c9

### PR DESCRIPTION
**Pull request by mxdvl**
## What does this change?

Add a badge for the 2022 Football World Cup.

World Cup 2022 Badge

As above, but a actually black!

## Does this change need to be reproduced in dotcom-rendering ?

- [X] No – It will trickle through automatically
- [ ] Yes

## Screenshots

| Before      | After      |
|-------------|------------|
| ![before][] | ![after][] |

[before]: https://user-images.githubusercontent.com/76776/201933717-e2f6b985-8088-4c19-afa2-0108f7950907.png
[after]: https://user-images.githubusercontent.com/76776/201936665-6ad27ff8-de6d-49f3-938d-453a27c753c5.png

## What is the value of this and can you measure success?

Resolves #25674

### Accessibility test checklist

- [ ] [Tested with screen reader](https://accessibility.gutools.co.uk/testing/web/screen-readers/)
- [ ] [Navigable with keyboard](https://accessibility.gutools.co.uk/testing/web/keyboard-navigation/)
- [X] [Colour contrast passed](https://accessibility.gutools.co.uk/testing/web/colour-contrast/)

### Tested

- [X] Locally
- [ ] On CODE (optional)

### CODE REVIEW

**Review by HarryFischer — APPROVED** [review]
Cheers @mxdvl  – can we hold off merging until tomorrow?

**Comment by mxdvl** [comment]
> Cheers @mxdvl – can we hold off merging until tomorrow?

Merge this when it suits best, @HarryFischer

**Pull request merged**
